explainer.md

```diff
@@ -47,7 +47,7 @@ const bufferB = new Float32Array(4).fill(0.8);
 const bufferC = new Float32Array(4);
 const inputs = {'A': bufferA, 'B': bufferB};
 const outputs = {'C': bufferC};
-graph.compute(inputs, outputs);
+context.compute(inputs, outputs);
```

Comment:
missing `graph` for compute?

Reply:
Fixed

Reply:
> As I https://github.com/webmachinelearning/webnn/pull/255#discussion_r816500166 in https://github.com/webmachinelearning/webnn/pull/255, I think it should support GPU as well.

I'll defer this to @RafaelCintron who insisted that GPU sync call is bad for user experience.